claragenomics/dl4atac/metrics.py

```diff
@@ -16,82 +16,72 @@
 from scipy.stats import rankdata
 
 class Metric(object):
-    """Metrics class"""
+    """Base metrics class."""
 
     def __init__(self):
         """Initialize class."""
-
         self.reset()
 
     def reset(self):
-        """Reset class."""
-
+        """Reset value of metric."""
         self.val = torch.tensor(0.).cuda() if torch.cuda.is_available() else torch.tensor(0.)
 
     def get(self):
-        """Get class."""
-
+        """Get metrics value."""
         return self.val
 
     def __str__(self):
-        """Return string."""
-
+        """Serialize metric into a string."""
         return "{}:{:7.4f}".format(type(self).__name__.lower(), self.val)
 
     def __call__(self, x, y):
-        """
-        Callable object.
+        """Call the metric class as a function.
 
         Args:
-            x: 
-            y: 
-    
+            x: Value 1
```

Comment:
What is Value 1 and Value 2 ? A list or an int ..?

Reply:
Updated